src/core_modules/capture-core/components/Pages/Search/SearchDomainSelector/SearchDomainSelector.js

```diff
@@ -85,4 +86,4 @@ export const Index =
       </Section>
       );
 
-export const SearchDomainSelector = withStyles(styles)(Index);
+export const SearchDomainSelector: ComponentType<OwnProps> = withStyles(styles)(Index);
```

Comment:
Without this bit here flow will lose the props that we the component has been annotated with before wrapped wit the HOC. And that goes for any HOC.